### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the action performed when "Alt+B" shortcut key is used in "push Image" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1470]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Push Image.
4. In "Push Image" dialog, use "Alt +B" to expand/collapse the Before launch expand/collapse button
5. Observe the behaviour of Jaws and NVDA 

**Actual Result:**
Jaws and NVDA are not announcing the performed action when Alt+B shortcut key is used

**Expected Result:**
Jaws and NVDA should announce the performed action when short cut key is used
Jaws and NVDA should announce as
Before Launch :build Artifacts ,Activate tool window expanded Alt+B

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-expand-button](https://user-images.githubusercontent.com/22956130/36102907-c35e1424-1033-11e8-9032-97825dc2fd09.jpg)
![982fb639](https://user-images.githubusercontent.com/22956130/36102911-c6ce6cee-1033-11e8-8f12-56484db702eb.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749922
